tools/README.md

```diff
@@ -0,0 +1,3 @@
+# Tools
+
```

Comment:
Please add the `## Overview` section. You can base on the Overview documents of [components](https://github.com/kyma-project/kyma/blob/master/components/README.md) and [tests](https://github.com/kyma-project/kyma/blob/master/tests/README.md). For the time being, you do not have to elaborate on the tools readme that much.